examples/install_vars.yaml

```diff
@@ -10,10 +10,12 @@ storage_type: nfs
 log_level: info
 release_image_override: ""
 
+node_connection_timeout: 2700
+
 rhcos_kernel_options: []
+sysctl_tuned_options: false
```

Comment:
hmm.. why this entry for timeout option ?

Reply:
Just added the missing variable in example.

Reply:
yeah.. better to split this in a new PR and you can put the message as adding missing variable in examples

Reply:
ok